browser/src/ShortTandemRepeatPage/ShortTandemRepeatAttributes.tsx

```diff
@@ -45,6 +42,39 @@ const ShortTandemRepeatRepeatUnits = ({ shortTandemRepeat }: ShortTandemRepeatRe
     )
   }
 
+  if (
+    (repeatUnitsByClassification as any).pathogenic &&
+    (repeatUnitsByClassification as any).pathogenic.length === 1 &&
+    !(repeatUnitsByClassification as any).benign &&
+    !(repeatUnitsByClassification as any).unknown
```

Comment:
Just wanted to double check on the casting as `any`s here. 

I see that `repeatUnitsByClassification` is now of type `Record<string, string[]>` with this PR. Some, but not all of the types have been tightened as result of this. Notably, there are ~10 `(repeatUnitsByClassification as any)` casts in this file.

Can the `repeatUnitsByClassification` type be tightened more to prevent the need for some of these casts?

No worries if not, just wanted to check on this since the `any`s stood out, and wanted to ask for my nutritional benefit.